api/nakadi-event-bus-api.yaml

```diff
@@ -548,6 +548,47 @@ paths:
           description: |
             Schema object.
 
+  '/event-types/{name}/cursor-distances':
+    post:
+      tags:
+        - unmanaged-api
+        - monitoring
+        - management-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        Calculate the distance between two offsets. This is useful for performing checks for data completeness, when
+        a client has read some batches and wants to be sure that all delivered events have been correctly processed.
+      parameters:
+        - name: name
+          in: path
+          description: Name of the EventType
+          type: string
+          required: true
+        - name: cursors-distances-query
+          in: body
+          description: |
+            List of pairs of cursors: `initial_cursor` and `final_cursor`. Used as input by Nakadi to
+            calculate how many events there are between two cursors. The distance doesn't include the `initial_cursor`
+            but includes the `final_cursor`. So if they are equal the result is zero.
+          schema:
+            type: array
+            items:
+              $ref: "#/definitions/CursorDistanceQuery"
+          required: true
+      responses:
+        '200':
+          description: OK
+          schema:
+            type: array
```

Comment:
Please consider following the guideline to use objects for top level data structures (http://zalando.github.io/restful-api-guidelines/compatibility/Compatibility.html#must-always-return-json-objects-as-toplevel-data-structures-to-support-extensibility) to enable future compatible extensions. Even though it is sometimes hard to imagine the use cases, the rule is based on long term experience about managing API changes.